src/backend/core/models.py

```diff
@@ -643,6 +671,12 @@ class Team(BaseModel):
     Team `service_providers`.
     """
 
+    # Allow up to 80 nested teams with 62^5 (916_132_832) root nodes
+    alphabet = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
+    # Django treebeard does not allow max_length = None...
+    steplen = 5
+    path = models.CharField(max_length=5 * 80, unique=True, db_collation="C")
```

Comment:
I was wondering if this collation impacts the use of "startswith" :thinking:

Reply:
> Using the C collation can improve database performance for certain operations, like sorting and comparison, because it uses simple byte-wise comparison rather than more complex locale-aware rules. However, this comes at the cost of losing locale-specific behaviors, such as case insensitivity and proper sorting for different languages.

So I guess it's only better :) Actually it still works properly in the specific case of "startswith" but I can't tell for the performance improvement or not.